static/src/javascripts/lib/closest.js

```diff
@@ -0,0 +1,24 @@
+// @flow
+/**
+ * Adds a wrapper around Element.closest()
+ */
+
+function closest(element: ?Element, selectors: string): ?Node {
+    if (!element) return null;
+
+    if ('closest' in Element.prototype) {
+        return element.closest(selectors) || null;
+    }
+
+    let el = element.parentElement;
```

Comment:
Yeah, you have to think in pure logic, where the closest element of an element is the element itself. My initial polyfill had a bug where I used `parentNode` instead of `parentElement`, but I see you are using it, so good (although no need to test for `nodeType` as `Element.parentElement ≡ Element.parentNode ^ Element.parentNode.nodeType = 1`).